configure.py

```diff
@@ -2393,8 +2410,7 @@ def release_date(datestamp):
 if __name__ == '__main__':
     try:
         main()
-    except Exception as e:
-        import traceback
+    except ConfigureError as e:
```

Comment:
Capturing and logging the stack trace regardless of what happened during `main` is exactly what this top level block is for.